content/300-guides/200-deployment/201-serverless/index.mdx

```diff
@@ -11,6 +11,6 @@ If your application is deployed via a Function-as-a-Service (FaaS) offering and
 
 </TopBlock>
 
-## Serverless (FaaS) guides
+## Guides for Function-as-a-Service providers
```

Comment:
I might have potentially said something other in the past, but this should probably say "Serverless Functions" somehow.